### PR TITLE
can now view books on the cards

### PR DESCRIPTION
**Pull request by diddydeeable**
## CORS implements so you can access the backend data from any origin begining with `localhost:`

Screenshot 2024-06-18 at 12 43 12

# how to test

1. run backend `npm run dev`
2. run frontend npm run dev`

go onto front end web address    e.g. `http://localhost:5173/`

### CODE REVIEW

**Review by halimahexe — APPROVED** [review]
LGTM

File: src/pages/Product/Product.tsx [hunk 1]
```diff
@@ -1,25 +1,49 @@
...
+	if (!response.ok) {
+		throw new Error(`HTTP error! status: ${response.status}`);
+	}
+
+	console.log(response);
+	const data = await response.json();
+	return data;
+}
+
+function ProductList() {
```
Comment (halimahexe) [inline src/pages/Product/Product.tsx]: As you've changed this name, I think the component folder and file name should be updated to `ProductList` too to match convention.

**Review by halimahexe — CHANGES_REQUESTED** [review]
Please can you make the change listed below?

**Review by oskarprzybylski23 — APPROVED** [review]
Good job!

**Review by halimahexe — APPROVED** [review]
LGTM

**Pull request merged**